src/module/admin/enroll-user.ts_old

```diff
@@ -1,15 +1,15 @@
+/* import { POINT_CONVERSION_UNCOMPRESSED } from 'constants';
```

Comment:
probably we could even remove this file. 🤔 
This `_old` postfix comes from the fabric_2.2 update branch.